src/plugins/midi-piano/midi-piano-editor.js

```diff
@@ -0,0 +1,197 @@
+  };
+
+  const handleExternalSourceUrlValueChanged = event => {
+    const { value } = event.target;
+    changeContent({ sourceUrl: value });
+  };
+
+  const handleInternalSourceUrlValueChanged = event => {
+    const { value } = event.target;
+    changeContent({ sourceUrl: value });
+  };
+
+  const handleInternalSourceUrlFileNameChanged = value => {
+    changeContent({ sourceUrl: value });
+  };
+
+  const handleMidiTrackTitleValueChanged = event => {
+    const { value } = event.target;
+    changeContent({ midiTrackTitle: value });
+  };
+
+  const handleActiveNotesChanged = midiValue => {
+    let validNoteRange;
+    let array = [...activeNotes];
+
+    if (array.includes(midiValue)) {
+      const index = array.indexOf(midiValue);
+      array.splice(index, 1);
+      if (array.length === 2) {
+        array.sort((a, b) => {
+          return a - b;
+        });
+        validNoteRange = {
+          first: array[0],
+          last: array[1]
+        };
+        changeContent({ activeNotes: array, noteRange: validNoteRange });
+        return;
+      }
+      changeContent({ activeNotes: array });
+      return;
+    }
+    array = [...activeNotes, midiValue];
+    if (array.length === 2) {
+      array.sort((a, b) => {
+        return a - b;
+      });
+      validNoteRange = {
+        first: array[0],
+        last: array[1]
+      };
+      changeContent({ activeNotes: array, noteRange: validNoteRange });
+      return;
+    }
+    changeContent({ activeNotes: array, noteRange: validNoteRange });
+  };
+
+  const renderSourceTypeInput = (value, onChangeHandler) => (
+    <FormItem label={t('common:source')} {...formItemLayout}>
+      <RadioGroup value={value} onChange={onChangeHandler}>
+        <RadioButton value={MIDI_SOURCE_TYPE.external}>{t('common:externalLink')}</RadioButton>
+        <RadioButton value={MIDI_SOURCE_TYPE.internal}>{t('common:internalCdn')}</RadioButton>
+      </RadioGroup>
+    </FormItem>
+  );
+
+  const renderInternalSourceTypeInput = (value, onInputChangeHandler, onFileChangeHandler) => (
+    <FormItem label={t('common:internalUrl')} {...formItemLayout}>
+      <div className="u-input-and-button">
+        <Input
+          addonBefore={CDN_URL_PREFIX}
+          value={value}
+          onChange={onInputChangeHandler}
+          />
+        <ResourcePicker
+          url={storageLocationPathToUrl(value)}
+          onUrlChange={url => onFileChangeHandler(urlToStorageLocationPath(url))}
+          />
+      </div>
+    </FormItem>
+  );
+
+  const renderExternalSourceTypeInput = (value, onChangeHandler) => (
+    <FormItem label={t('common:externalUrl')} {...formItemLayout} {...validation.validateUrl(value, t)} hasFeedback>
+      <Input value={value} onChange={onChangeHandler} />
+    </FormItem>
+  );
+
+  const renderMidiTrackTitleInput = (value, onChangeHandler) => (
+    <FormItem label={t('midiPiano:midiTrackTitle')} {...formItemLayout} hasFeedback>
```

Comment:
Because the translation here is instantiated like this `const { t } = useTranslation('midiPiano');` you don't need to specify the "midiPiano" namespace anymore, as it is the current namespace.
This means you can write for example `t('midiTrackTitle')}` and only keep the namespace when it's a different one (like for `t('common:externalUrl')` for example)